.github/workflows/maven-verify.yml

```diff
@@ -25,5 +25,9 @@ jobs:
   build:
     name: Verify
     uses: apache/maven-gh-actions-shared/.github/workflows/maven-verify.yml@v2
+    with:
+      ff-jdk: '17'
+      maven_version: 3.8.5
+      jdk-matrix: '[ "17" ]'
```

Comment:
Why only 17?
You can  exclude only jdk 8

```
 matrix-exclude: >
        [  {"jdk": "8"} ]
```

Reply:
See here https://github.com/apache/maven-indexer/pull/207#issuecomment-1115316420